### DISCUSSION TITLE
Renewing of teacher account and location information

### DISCUSSION THREAD

**Discussion by stefan11** [community/community#46350] [GitHub Education]
Hi,

I want to renew my github teacher account. I can log in and specify details, but when I wont to go on, the continue button is greyed out and it says "To continue you must modify your browser settings to allow GitHub to use your location information." I did this with two different browsers, no change.

Any help would be appreciated.

Best

**Comment by jge162** [comment]
Hello @stefan11 please try the following below.
 
- Allow location access in your browser settings.
- Try using a different browser, for example, switch from Google Chrome to Mozilla Firefox or vice versa.
- Disable any browser extensions or plugins that might block location access.
- Clear your browser cache and cookies.
- Restart your browser and try again.

If this still does not work. You can
Reach out to GitHub [Education support](https://support.github.com/request/education?tags=hubberfy_education)
